api/HTMLTableHeaderCellElement.json

```diff
@@ -0,0 +1,229 @@
+{
+  "api": {
+    "HTMLTableHeaderCellElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLTableHeaderCellElement",
+        "support": {
+          "webview_android": {
+            "version_added": false,
+            "notes": "Members of the HTMLTableHeaderCellElement interface now implemented on HTMLTableCellElement."
```

Comment:
This note seems good, but I think it would be a little more readable with `HTMLTableHeaderCellElement` and `HTMLTableCellElement` in `<code>` tags